README.md

```diff
@@ -51,6 +51,37 @@ npm install --save ts-json-schema-generator
     Do not sort properties.
 ```
 
+## Programmatic Usage
+
+Here is an example of using it programmatically:
+
+```typescript
+import { SchemaGenerator } from "ts-json-schema-generator";
+import * as ts from "typescript";
+import { createFormatter } from "ts-json-schema-generator/dist/factory/formatter";
+import { createParser } from "ts-json-schema-generator/dist/factory/parser";
+import { createProgram } from "ts-json-schema-generator/dist/factory/program";
+import { Config } from 	"ts-json-schema-generator/dist/src/Config";
```

Comment:
We should instead expose these from the index.js file so you don't have to import from specific files.

Reply:
That would be good. It does work like this now.

Reply:
Let's make it work so that the usage is clean.

Reply:
We are exposing all of this from `factory/index.ts`.